sonar-ws/src/main/java/org/sonarqube/ws/client/qualityprofile/QualityProfileWsParameters.java

```diff
@@ -23,11 +23,27 @@
 
   public static final String CONTROLLER_QUALITY_PROFILES = "api/qualityprofiles";
 
+  public static final String ACTION_ACTIVATE_RULE = "activate_rule";
+  public interface ActivateActionParameters {
+    String PARAM_PROFILE_KEY = "profile_key";
+    String PARAM_RULE_KEY = "rule_key";
+    String PARAM_SEVERITY = "severity";
+    String PARAM_RESET = "reset";
+    String PARAM_PARAMS = "params";
+  }
+
+  public static final String ACTION_RESTORE = "restore";
+  public interface RestoreActionParameters {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1214)